### PR TITLE
storybook args and props edited

### PR DESCRIPTION
**Pull request by MarcinkEve**

### CODE REVIEW

**Review by INKR1 — APPROVED** [review]

File: src/Components/StoryBookExampleComponent/StoryBookExampleComponent.stories.jsx [hunk 1]
```diff
@@ -1,124 +1,123 @@
-import { useState } from 'react'
```
Comment (prodevshub) [inline src/Components/StoryBookExampleComponent/StoryBookExampleComponent.stories.jsx:1]: Why eslint removing single qoutes? It should not do that. We using single qoutes in this project.

File: .storybook/readme.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+```javascript
+export const Primary = Template.bind({});
+Primary.decorators = [
+  (Story) => (
+    <div style={{ margin: '3em' }}>
+      <Story />
+    </div>
+  ),
+];
+```
```
Comment (prodevshub) [inline .storybook/readme.md]: package-lock.json file shouldn't be changed if you didn't change anything in package.json

File: .storybook/readme.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# Install:
```
Comment (prodevshub) [inline .storybook/readme.md]: I think this file should be named storybook.md, and kept in root directory.

File: src/Components/StoryBookExampleComponent/StoryBookExampleComponent.jsx [hunk 4]
```diff
@@ -3,9 +3,7 @@ import logo from '../../logo.svg'
 import '../../App.css'
 
 // cia priimam propsus i komponenta is stories
-const StoryBookExampleComponent = ({
-    name, footer, padding, bgColor, handleOnChange, color, backgroundColor, fontSize, fontWeight, textAlign, primary, value, label
-}) => (
+const StoryBookExampleComponent = ({ primary, label, color, backgroundColor, padding, fontSize, fontWeight, textAlign, margin, footer, name }) => (
```
Comment (prodevshub) [inline src/Components/StoryBookExampleComponent/StoryBookExampleComponent.jsx]: Please keep same eslint rules as we have so far, and make new PR wihtout style changes.

**Review by prodevshub — CHANGES_REQUESTED** [review]

File: storybook.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
+# Install:
```
Comment (MarcinkEve) [inline storybook.md:1]: Moved to root directory

File: src/Components/StoryBookExampleComponent/StoryBookExampleComponent.stories.jsx (hunk 1 above)
Comment (MarcinkEve) [reply src/Components/StoryBookExampleComponent/StoryBookExampleComponent.stories.jsx:1]: I've found out, that in code editor eslint wasn't turned on..

File: .storybook/readme.md (hunk 3 above)
Comment (MarcinkEve) [reply .storybook/readme.md]: Done

File: .storybook/readme.md (hunk 2 above)
Comment (MarcinkEve) [reply .storybook/readme.md]: I didn'y remember if I changed anything here... Here was a conflict, I probably choose the wrong solution...

File: src/Components/StoryBookExampleComponent/StoryBookExampleComponent.jsx (hunk 4 above)
Comment (MarcinkEve) [reply src/Components/StoryBookExampleComponent/StoryBookExampleComponent.jsx]: Style restored

**Review by prodevshub — APPROVED** [review]

**Pull request merged**